.github/issue_template.md

```diff
@@ -0,0 +1,39 @@
+<!--
```

Comment:
Yes, every time you try to open an issue, this will print in the issue text section